docs/slurk_prerequisites.rst

```diff
@@ -92,13 +101,13 @@ Install docker into Ubuntu
 
     $> curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
 
-4. Use the following command to set up the stable repository.
+4. Use the following command to set up the stable repository:
```

Comment:
For installing docker better link to the [official documentation](https://docs.docker.com/get-docker/).